server/svix-server/tests/utils/mod.rs

```diff
@@ -248,20 +258,24 @@ impl TestReceiver {
         let (tx, data_recv) = mpsc::channel(32);
         let (header_tx, header_recv) = mpsc::channel(32);
 
+        let response_status_code = Arc::new(Mutex::new(ResponseStatusCode {
+            status_code: resp_with,
+        }));
+
+        let routes = axum::Router::new()
+            .route(
+                "/",
+                axum::routing::post(test_receiver_route).get(test_receiver_route),
+            )
+            .layer(axum::extract::Extension(tx))
+            .layer(axum::extract::Extension(header_tx))
+            .layer(axum::extract::Extension(response_status_code.clone()))
+            .into_make_service();
```

Comment:
This is an unrelated change that makes the diff annoying, move it back to where it was.

Reply:
I did not just do this, the compiler complains about response_status_code because the closure is `move`. I suppose I could clone response_status_code to another declared variable outside the closure and use that inside it.

I think this way looks better in the code at the end of the day, that way looks better for the diff.

Reply:
OK.